src/components/ReleaseCard/index.jsx

```diff
@@ -95,32 +101,35 @@ function ReleaseCard(props) {
         }
         subheader={release.product}
         action={
-          <Tooltip title="History">
-            <IconButton>
-              <HistoryIcon />
-            </IconButton>
-          </Tooltip>
+          <div>
+            <FormControlLabel
+              control={
+                <Switch
+                  checked={release.read_only}
+                  onChange={handleAccessChange}
+                />
+              }
+              label="Read only"
```

Comment:
I think we should change this text somehow. `Read only` sounds like a statement to me (as in, this release is read only, even if it's unchecked). If the label text changed with the state that would work, or if it was more clearly a label. I'm open to other ideas too.

Reply:
> or if it was more clearly a label.

Perhaps something like:

![Screen Shot 2019-07-12 at 11 03 40 AM](https://user-images.githubusercontent.com/3766511/61138087-c6447100-a494-11e9-86ad-a48423b9cd2f.png)

Reply:
I picked a switch because a switch usually means to toggle the state of a setting. Once toggled to the left, the actions of that card become clickable.

Reply:
Let me know if you still don't like the current approach. I can try to think of another alternative.

Reply:
Yeah, I like the switch control, the text just threw me a bit. I like the screenshot above. Grouping the switch and the text together clears up the ambiguity I had.